src/main/java/uk/gov/justice/digital/hmpps/csr/api/dto/DetailDto.kt

```diff
@@ -57,14 +57,8 @@ DetailDto @JsonCreator constructor(
         actionType = detail.actionType?.let { type -> ActionType.from(type) }
       )
 
-    fun from(detail: CmdNotification): DetailDto {
-
-      // Note this is intended to be temporary - we are not expecting layer to be anything other than -1
-      // but would be nice to know
-      if (detail.layer != -1) {
-        log.warn("DetailDto.from: Found ${detail.staffId}, ${detail.levelId}, ${detail.onDate} with layer ${detail.layer}")
-      }
-      return DetailDto(
```

Comment:
layer -1 now filtered by triggers. this value means the current data, rather than historical ( >= 0 )